dockstore-webservice/src/main/resources/openapi3/openapi.yaml

```diff
@@ -4225,7 +4343,7 @@ paths:
           description: default response
       summary: Successful response if elastic search is up and running
       tags:
-        - metadata
```

Comment:
ok, now i'm confused which yq is supposed to be used

Reply:
I installed 3.1.2, which was the same as the circle ci config :( Any ideas?

Reply:
so in the last PR, it looks like the it was supposed to be 3.1.2.  now it looks like the new version (the one mentioned in develop)

Reply:
Should I run again with yq 3.3.2 (from develop)?

Reply:
It's worth a shot, or at least 3.3.1 looks like my install says this
```
$ snap info yq
name:      yq
summary:   A lightweight and portable command-line YAML processor
publisher: Mike Farah (mikefarah)
store-url: https://snapcraft.io/yq
contact:   https://github.com/mikefarah/yq/issues
license:   unset
description: |
  The aim of the project is to be the jq or sed of yaml files.
commands:
  - yq
snap-id:      b1xa1ED1Aw4HN9BnJVP3Je95pyEVN6gu
tracking:     latest/stable
refresh-date: 60 days ago, at 21:13 EDT
channels:
  latest/stable:    3.3.1 2020-06-11 (736) 2MB -
  latest/candidate: ↑                          
  latest/beta:      ↑                          
  latest/edge:      3.3.2 2020-06-15 (772) 2MB -
installed:          3.3.1            (736) 2MB -


```

Reply:
Also https://github.com/dockstore/dockstore/pull/3629

Reply:
Ok, I used 3.3.2 (same as develop)